week-1/2-mandatory/1-syntax-errors.js

```diff
@@ -2,20 +2,25 @@
 
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age){
+    //return "Hello, my name is " + name + "and I am " + age + " years old";
+    let myName = "Hello, my name is " + name + " and I am " + age + " years old";
+    return myName;
+}
 
 function getAddition(a, b) {
```

Comment:
One test fails `ReferenceError: getRemainder is not defined`. Can you figure what to fix here?